output_3.5.txt

```diff
@@ -0,0 +1,30 @@
+
+Question 3.5
+What are _id and borough of �Seafood� (cuisine) restaurants  which received at least one �B� grade in period from 2014-02-01 to 2014-03-01? Use projection to include only _id and borough.
```

Comment:
query is correct, answer is correct, but structure of answer is not correct. Pay attention to the task